### DISCUSSION TITLE
Filter and show specific columns

### DISCUSSION THREAD

**Discussion by Insperio** [area17/twill#854] [Q&A]
I was just wondering if it's possible to show different set of columns based on which filter is selected ?
So for example when showing "active" items we show creation time and for "disabled" items, we show last updated time ? 

I think it could be pretty useful if possible. I have one table with 20 possible columns and it feels a bit messy at the moment.

**Comment by ifox** [comment]
You should be able to do this by altering the index items depending on the active filters. I'd suggest you ask this type of question on our Spectrum community as this is exactly the type of use case many of them have been able to help with, and this is not a bug but rather a lack of fleshed out documentation. Do not hesitate to check the source code to understand the options you might not be thinking of because you are not seeing them in the docs.
